player.js

```diff
@@ -0,0 +1,58 @@
+Scrabble = require('./scrabble');
+var scrabble = new Scrabble();
+
+var Player = function(name){
+  this.plays = [];
+  this.name  = name;
+
+  this.play = function(word) {
+    if (this.hasWon === true) {
+      return false;
+    }
+    this.plays.push(word);
+    return this.plays;
+  };
+
+  this.total_score = function() {
+    var sum = 0;
+    for(var word of this.plays){
+      sum = sum + scrabble.score(word);
+    }
+    return sum;
+  };
```

Comment:
consider keeping ```this.total_score``` as a variable so you don't have to recalculate it each turn from scratch